dask/bytes/tests/test_s3.py

```diff
@@ -193,32 +201,32 @@ def test_getsize(fmt):
 
 def test_modification_time_read_bytes():
     with s3_context('compress', files) as s3:
-        _, a = read_bytes('compress/test/accounts.*', s3=s3)
-        _, b = read_bytes('compress/test/accounts.*', s3=s3)
+        _, a = read_bytes('compress/test/accounts.*', storage_options={'s3': s3})
+        _, b = read_bytes('compress/test/accounts.*', storage_options={'s3': s3})
```

Comment:
I suggest users write

```python
storage_options = {'s3': s3}
read_csv(..., storage_options=storage_options)
```

Passing storage options inline with parser options in not great at all from my opinion.